azure.go

```diff
@@ -391,3 +361,19 @@ func ConfigurationFromName(name string) (cloud.Configuration, error) {
 
 	return c, nil
 }
+
+func graphURIFromName(name string) (string, error) {
+	configs := map[string]string{
+		"AZURECHINACLOUD":        azureChinaCloudBaseURI,
```

Comment:
it might be worthwhile to make these constants as well

Reply:
Good idea. Done in https://github.com/hashicorp/vault-plugin-auth-azure/pull/121/commits/2b2bf18974cc94962f388ff8a574c9579925dd90.